### PR TITLE
/static/admin is created by collectstatic

### PR DESCRIPTION
**Pull request by bfirsh**
If they've got static_root, they'll have static_root/admin too

### CODE REVIEW

**Comment by bfirsh** [comment]
Added the new location for ../data too

File: settings/base.py [hunk 1]
```diff
@@ -17,7 +18,7 @@ SITE_ID = 1
 USE_I18N = True
 USE_L10N = True
 
-MEDIA_ROOT = PROJECT_DIR.parent.child('data')
```
Comment (idan) [inline settings/base.py:20]: This is awesome, but epio-specific. The right place for it is in epio.py, not base.py

Comment (bfirsh) [inline settings/base.py:20]: Aw snap, sorry! Trust me to not to think/test before committing.

File: epio.ini [hunk 2]
```diff
@@ -9,7 +9,6 @@ redis = true
 [static]
 /static = static_root
 /media = ../data
-/static/admin = ../shortcuts/django-admin-media
```
Comment (idan) [inline epio.ini:12]: This line is not needed because it's automatically configured as such?

Comment (bfirsh) [inline epio.ini:12]: It's created by collectstatic :)

**Pull request merged**